src/components/calcite-dropdown-item/calcite-dropdown-item.scss

```diff
@@ -5,45 +5,43 @@
 
 // ie11
 :host-context([scale="s"]) {
-  --calcite-dropdown-item-padding: #{$baseline/5 $baseline/1.5 $baseline/5 $baseline * 1.5};
+  @apply text--2 pr-3 pl-6 py-2;
 }
 
 :host-context([scale="m"]) {
-  --calcite-dropdown-item-padding: #{$baseline/3 $baseline/1.5 $baseline/3 $baseline * 1.5};
+  @apply text--1 pr-4 pl-8 py-3;
 }
 
 :host-context([scale="l"]) {
-  --calcite-dropdown-item-padding: #{$baseline/2 $baseline/1.5 $baseline/2 $baseline * 1.5};
+  @apply text-1 pr-5 pl-10 py-4;
 }
 
 :host-context([dir="rtl"][scale="s"]) {
-  --calcite-dropdown-item-padding: #{$baseline/5 $baseline * 1.5 $baseline/5 $baseline/1.5};
+  @apply text--2 pr-6 pl-3 py-2;
```

Comment:
you probably don't need the `text` or `py` elements in the `rtl` styles (only need to override things that will flip in rtl

Reply:
Thanks @paulcpederson.
Good catch.